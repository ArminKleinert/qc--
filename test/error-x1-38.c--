
section "foo" {

        label:
        bits32[2];

        foo () {
            goto label;
        }
}
