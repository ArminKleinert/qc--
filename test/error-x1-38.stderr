file error-x1-38.c--, line 8, characters 13-24 Error: goto-expression does not evaluate to a code pointer
